Vsix/CodeConversion.cs

```diff
@@ -308,7 +308,8 @@ private async IAsyncEnumerable<ConversionResult> ConvertProjectUnhandledAsync<TL
         var projectsByPath =
             _visualStudioWorkspace.CurrentSolution.Projects.ToLookup(p => p.FilePath, p => p);
 #pragma warning disable VSTHRD010 // Invoke single-threaded types on Main thread - ToList ensures this happens within the same thread just switched to above
-        var projects = selectedProjects.Select(p => projectsByPath[p.FullName].First()).ToList();
+        var projects = selectedProjects.SelectMany(p => projectsByPath[p.FullName]).ToList();
```

Comment:
This was the initial spike that allowed getting the project that's created for the open editor. I didn't pursue this since even with the faff of needing the windows open, it didn't seem to easily have access to the mapped spans to insert the code back into the original vbhtml file from the generated file